pkg/ui/static/js/graph.js

```diff
@@ -390,6 +393,9 @@ Prometheus.Graph.prototype.getOptions = function() {
       }
     }
   });
+
+  options.max_source_resolution = self.queryForm.find("select[name=max_source_resolution_input]").val();
```

Comment:
Maybe `select[name=max_source_resolution_input]` should be moved out somewhere because we are repeating ourselves?

Reply:
Refactored the 3 occurrences as `maxSourceResolutionInput`.